### ISSUE TITLE
ParseError when option --day is used

### ISSUE THREAD

**Issue by reaseno** [splash-cli/splash-cli#59]
```
× { ParseError: Unexpected token < in JSON at position 0 in "https://lambda.splash-cli.app/":
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <m...
    at stream.catch.then.data (C:\Users\...\AppData\Roaming\npm\node_modules\splash-cli\node_modules\got\index.js:380:15)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  name: 'ParseError',
  host: 'lambda.splash-cli.app',
  hostname: 'lambda.splash-cli.app',
  method: 'GET',
  path: '/',
  protocol: 'https:',
  url: 'https://lambda.splash-cli.app/',
  statusCode: 200,
  statusMessage: 'OK' }
× Unable to connect.
```

**Comment by Rawnly** [comment]
I'm working on a fix! I'm releasing it in a few minutes ✌️

**Comment by Rawnly** [comment]
Fixed in [v3.4.6](https://github.com/splash-cli/splash-cli/releases/tag/3.4.6)

**Comment by reaseno** [comment]
Workening 👍
